### ISSUE TITLE
"fungal sorus development" versus "spore-bearing structure development". 

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#13078]
Hello @dianeoinglis

Right now in the ontology we have "fungal sorus development" and "spore-bearing structure development". 

The logical definition of "spore-bearing structure development" states that "anatomical structure development' and ('results in development of' some sporophore)". Is a sporophore the same as a fungal sorus? If so, these terms should be merged. If not, which term should be a child of the other or should they remain siblings. 

@pfey03 @dosumis

**Comment by dianeoinglis** [comment]
Hi, apologies for the delayed response. On behalf of the entire fungal research community, I apologize for the mess that has been made with terms used inaccurately by some groups and identical terms used for different things in different species for many decades. Fungal researchers have only recently begun to use precise and accurate terms and the use of the GO is partly impetus. Only an expert in conidial fungi, pathogenic fungi, dimorphic and non-dimorphic fungal species that has been exposed to talks and papers in numerous fungal can begin to understand the distinctions, nuances, incorrect and correct or purely historical uses of terms. It's baaaad.

A "fungal sorus" (pl: sori) is a type of reproductive structure that consists of sporangia (an enclosure in which clusters or masses of spores are produced) and is characteristic of some fungi, particularly rusts and smuts. 

A "sporangium" (pl: sporangia) is an **enclosure**, consisting of one or more cells, in which spores are borne/produced. 

I think we need some clarification of the definitions and the online definitions do not make the issue clear. Let me make a distinction and see what you think. 

A fungal sorus is a reproductive structure that forms for the purpose of producing spores. A sorus can also be considered a type of fruiting body that consists of a sporangium (enclosure) in which the spores are actually generated. Technically speaking, a "spore bearing structure" develops with the outcome of producing sporogenous cells within (or upon) the structure. The spores are actually produced by the sporogenous cells that develop in/on the sporophore. Not all sporophores are sori.

The details for a commonly annotated species: a conidiophore (spore-bearing structure) of Aspergillus species consists of several cell types that form on/from the latter cell-type. In order of development/production from a hypha that has acquired competence to reproduce by sporulation/conidiation, 1) a foot cell, 2) a vesicle, 3) several metulae (sing: metula) except not A. fumigatus), 4) several phialides (sing: phialide), 5) then the spores are produced in chains (not sporangia) directly from each phialide. A conidiophore is not a sorus and do not produce spores within a sporangium. A conidiophore is an anatomical structure.

In Cryptocococcus, the "basidium" is the actual spore-producing cells within the hymenophore of a basidiocarp." The most common term you will find in the literature for this species is "fruiting body/fruitbody/fruit body," not any type of "basidiophore" or "basidiocarp" etc, though it would be technically correct. An expert fungal biologist/curator is needed to disambiguate.

The common definition of sporophore can be confusing but the Latin root for "phore" is "an agent or bearer of a specified thing" and to me that would refer to the actual sporogenous cell that directly produces the spores.

If these definitions and explanations do not help, let me know and I will attempt to further clarify.
Diane

**Comment by cmungall** [comment]
heh, these kinds of terminological nuances and clashes are common but sounds like the fungi people have a particularly tough problem!

Thanks for the clarifications. We'd like to use FAO as the reference, so do you think we can get your excellent definitions into FAO (and new term for 'sorus' etc)

**Comment by dianeoinglis** [comment]
Yes. I have a laptop problem that is hindering the actual edits to the FAO.obo. All the proposed terms with projected FAO IDs are in the FAO GitHub Issue Tracker. I think it might be best for curation purposes to make the more obscure terms synonyms of the terms in common use.

'Fruiting body,' identical with 'fruit body' and 'fruitbody' is quite coomonly used. A synonym is 'sporocarp.' Child terms of 'sporocarp' are 'basidiocarp' and 'ascocarp' in Basidiomycetes and Ascomycetes, respectively. 'Fungal sorus' is a less common synonym of the fruiting body. 'Sporocarp' is also best as a synonym as it's not used much in the commonly annotated fungal species. The prefix 'fungal' was added to distinguish slime-mold sorus which is not a reproductive structure. Fern also make sori and sporophores and these are similar biologically to fungal structures. Shall a new mini-project ticket or can these changes be made in this ticket?

I will update the issues, create definitions and assign IDs in the tracker. I can make the proposed changes in the tracker. For expediency, can someone with a functioning computer edit the FAO for now when I've made all edits-to-be very clear? I'm using a smartphone for GitHub comments.

**Comment by pgaudet** [comment]
Hi @cmungall 
Who in the FAO team could work on that ? 
Thanks, 
Pascale

**Comment by cmungall** [comment]
Any of us in GO are free to make PRs....

**Comment by pgaudet** [comment]
Good ! @dianeoinglis I'm willing to make the edits then. Let me know which issues this affects. 

Thanks, 

Pascale

**Comment by dianeoinglis** [comment]
Wonderful! If you haven't done so already, look at the issues in the tracker. It has NTR information, terms and defs to be modified, etc. There are quite a few updates to be made.

**Comment by pgaudet** [comment]
@ValWood @mah11 

Maybe you can advise on how to move forward ?

**Comment by mah11** [comment]
Well, I am far from the expert-in-many-fungi who might be able to appreciate all the angles @dianeoinglis mentioned, but with that caveat:

I have some thoughts about tidying up FAO as well as looking at its open tickets. The information in this ticket looks really useful for that. So it makes sense for me (and Antonia, once she gets involved) to do some work on FAO and then come back to this ticket.

I hope it's not urgent, because the chance that I'll have time this week or even next is vanishingly small. But I hope before the end of the month ...

One thing I can say now: "spore-bearing structure" terms must NOT be merged with corresponding "fungal sorus" terms. "Spore-bearing structure" absolutely has to be broader than fungal anything, for the simple reason that ferns and mosses also produce spores in/on specialized structures. (I don't know, but can try to find out, whether spore-bearing structures in ferns or mosses are called "sporangia" or "sporophores".)
